pkg/logs/logs.go

```diff
@@ -1,8 +1,90 @@
 package logs
 
 import (
+	"errors"
 	"log"
+	"log/slog"
 	"os"
+
+	"github.com/go-logr/logr"
+	"github.com/spf13/pflag"
+	cliflag "k8s.io/component-base/cli/flag"
+	"k8s.io/klog/v2"
+)
+
+var Log *log.Logger
+
+func init() {
+	Log = slog.NewLogLogger(slog.Default().Handler(), slog.LevelDebug)
+}
```

Comment:
I can start by just doing this, I think.

Then replacing the and ultimately removing the use of Log